isisd/isis_tlvs.c

```diff
@@ -4227,6 +4227,21 @@ static size_t isis_router_cap_tlv_size(const struct isis_router_cap *router_cap)
 	}
 #endif /* ifndef FABRICD */
 
+	if (router_cap->srv6_cap.is_srv6_capable) {
+		sz += 2 + ISIS_SUBTLV_SRV6_CAPABILITIES_SIZE;
+
+		nb_algo = isis_tlvs_sr_algo_count(router_cap);
+		if (nb_algo != 0)
+			sz += 2 + nb_algo;
```

Comment:
Do we want to use a define here for a magic value?

Reply:
Fixed.